### PR TITLE
Existing tips are all in markdown now.

### PR DESCRIPTION
**Pull request by prnadago**

### CODE REVIEW

File: HotTips/Tips/NAV_QuickFind.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Press **Ctrl+F** to quickly find in the current document.
+
+Press**Ctrl+H** to quickly replace in the current document.
```
Comment (justcla) [inline HotTips/Tips/NAV_QuickFind.md]: Need a space before the Ctrl+H shortcut.

**Comment by prnadago** [comment]
Done

**Comment by justcla** [comment]
You beat me! I was doing it right now. :-D

**Comment by justcla** [comment]
Thanks Pratik!

**Pull request merged**